bundles/org.openhab.binding.denonmarantz/src/main/feature/feature.xml

```diff
@@ -4,7 +4,6 @@
 
 	<feature name="openhab-binding-denonmarantz" description="Denon / Marantz Binding" version="${project.version}">
 		<feature>openhab-runtime-base</feature>
-		<feature>openhab-transport-mdns</feature>
 		<feature dependency="true">openhab.tp-jaxb</feature>
```

Comment:
Isn't this also redundant? I think we use JAXB for XML processing in core, too.

Reply:
Looks like both Jackson and JAX-B are always preinstalled.
The openhab.tp-jackson feature is used for Swagger/Marketplace/org.openhab.core.model.yaml.
openhab.tp-jaxb is required for the JAX-RS Whiteboard.
Shall I remove them both?

Reply:
Yes, I think so.